pyproject.toml

```diff
@@ -150,6 +150,7 @@ ignore = [
 include = [
     "charmcraft",
     "tests/unit",
+    "tests/integration",
     # "tests",  # TODO: enable tests
```

Comment:
is this TODO still valid, can we get rid of it now?

Reply:
There are legacy tests outside of those structures, but they can probably be left. TODO removed.